code/game/gamemodes/objectives_subtypes.dm

```diff
@@ -0,0 +1,73 @@
+/datum/objective/debrain/science
+	name = "Debrain science"
+	target_department = DEPARTMENT_SCIENCE
```

Comment:
instead of having a million overrides for murder/engineer, murder/science, murder/sec, etc. wouldnt it be better to have a proc on datum/antag_org that creates these objectives and overrides these variables?

Reply:
Yeah I'll rewrite how this works to shorten lines before merge